spec/unit/ConfigChanges.spec.js

```diff
@@ -57,24 +57,28 @@ describe('PlatformMunger', function () {
 
         it('should call parent\'s method with the same parameters', function () {
             munger.apply_file_munge(WINDOWS_MANIFEST, munge, false);
-            expect(BaseMunger.prototype.apply_file_munge).toHaveBeenCalled();
+            expect(BaseMunger.prototype.apply_file_munge).toHaveBeenCalledWith(WINDOWS_MANIFEST, munge, false);
         });
 
         it('should additionally call parent\'s method with another munge if removing changes from windows 10 appxmanifest', function () {
             munger.apply_file_munge(WINDOWS10_MANIFEST, munge, /*remove=*/true);
-            expect(BaseMunger.prototype.apply_file_munge).toHaveBeenCalled();
-            expect(BaseMunger.prototype.apply_file_munge).toHaveBeenCalled();
+            expect(BaseMunger.prototype.apply_file_munge).toHaveBeenCalledWith(WINDOWS10_MANIFEST, munge, true);
+            expect(BaseMunger.prototype.apply_file_munge).toHaveBeenCalledWith(WINDOWS10_MANIFEST, jasmine.any(Object), true);
```

Comment:
I know this is an original code but I wonder if the second check is actually checking something?